### DISCUSSION TITLE
User login (not bot) - 401: Unauthorized

### DISCUSSION THREAD

**Discussion by pmouraguedes** [Rapptz/discord.py#6393] [Q&A]
I'm trying to run the example reply.py using my normal user token (not the bot one). I'm getting 401: Unauthorized.

Is it currently possible to login as a normal user?

**Comment by yabberish** [comment]
You're not advised to login as a user, as it's against discord's terms of service, I suggest you don't try to run the bot with a user token.

**Reply by pmouraguedes** [reply]
Oh I see, thanks. 
I wanted to subscribe to channel messages on a server I don't own. Is this not possible to do?

**Reply by yabberish** [reply]
I'm not share, sorry about that.

But you're probably looking for `on_message`.

**Reply by bandwiches** [reply]
Bots can do it if they have permission to listen to the channel in the Guild, but they need to invite your bot to the guild first which is probably not what you're looking for.

**Reply by pmouraguedes** [reply]
Exactly, I don't own the guild so that is not gonna happen.
So, there is no way to read messages unless you're a bot and you are a guild admin?

**Reply by 15696** [reply]
You can. It's just not advised. The issue is that you have to pass False to the `bot` argument.

**Reply by bandwiches** [reply]
> You can. It's just not advised. The issue is that you have to pass False to the `bot` argument.

My mind is about to blow up - I think I'm following you here, but how does one pass `False` to a `Bot` argument and which argument would that be?

Wondering if it's possible to manually create a `Channel` passing `channel_id` and filter via `on_message()` for specified channel?

**Reply by 15696** [reply]
@bandwiches Not commands.Bot, but rather the `run` method.
```py
client.run("token", bot = False)
```
It's [not recommended](https://discordpy.readthedocs.io/en/latest/api.html#discord-models) to instance Discord models directly. Just get the channel with `client.get_channel`.

> The classes listed below are not intended to be created by users and are also read-only.

> For example, this means that you should not make your own User instances nor should you modify the User instance yourself.

**Reply by pmouraguedes** [reply]
Thanks :)
